<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInWithRegister</class>
 <widget class="QDialog" name="LogInWithRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>496</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>283</width>
    <height>518</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LogInWithRegister</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1f2833;
color: white;
border-radius: 5px;
QLineEdit {
	background-color: rgb(55, 71, 90)
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="allBox" native="true">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color: rgb(55, 71, 90)
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="loginPage">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="loginnameLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Logowanie</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="logemailLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>E-mail</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="logemailLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="logpasswdLabel">
            <property name="text">
             <string>Hasło</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="logpasswdLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="logerrLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: red;</string>
            </property>
            <property name="text">
             <string>Złe dane</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="registerPage">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="registernameLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Rejestracja</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="regnameLabel">
            <property name="text">
             <string>Imię</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="regnameLineEdit">
            <property name="maxLength">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="regsurnameLabel">
            <property name="text">
             <string>Nazwisko</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="regsurnameLineEdit">
            <property name="maxLength">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="regemailLabel_2">
            <property name="text">
             <string>E-mail</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="regemailLineEdit">
            <property name="maxLength">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="regpasswdLabel">
            <property name="text">
             <string>Hasło</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="regpasswdLineEdit">
            <property name="maxLength">
             <number>50</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="regprovinceLabel">
            <property name="text">
             <string>Wojewodztwo</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="regprovinceComboBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Dolnośląskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kujawsko-pomorskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Lubelskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Lubuskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Łódzkie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Małopolskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mazowieckie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Opolskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Podkarpackie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Podlaskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pomorskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Śląskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Świętokrzyskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Warmińsko-mazurskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wielkopolskie</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Zachodniopomorskie</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="regzipcodeLabel">
            <property name="text">
             <string>Kod pocztowy</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="zipcodeBox" stretch="4,3,4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="regzipcodeLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>15</width>
                <height>0</height>
               </size>
              </property>
              <property name="inputMask">
               <string>99-999</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="regcityLabel">
            <property name="text">
             <string>Miasto</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="regcityLineEdit">
            <property name="maxLength">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="regaddressLabel">
            <property name="text">
             <string>Adres</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="regaddressLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="regerrorLabel">
            <property name="styleSheet">
             <string notr="true">background-color: red;</string>
            </property>
            <property name="text">
             <string>Podany użytkownik już istnieje</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Zaloguj się</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons_white/log-in.png</normaloff>:/icons_white/log-in.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="registerButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Zarejestruj się</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons_white/register.png</normaloff>:/icons_white/register.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
